hack/common.sh

```diff
@@ -7,6 +7,9 @@ KUBEVIRT_ANSIBLE_DIR="$(
 OUT_DIR=$KUBEVIRT_ANSIBLE_DIR/_out
 TESTS_OUT_DIR=$OUT_DIR/tests
 
+FUNC_TEST_ARGS="${FUNC_TEST_ARGS:--test.timeout 60m --junit-output=exported-artifacts/tests.junit.xml}"
+CDI_TEST_ARGS="${CDI_TEST_ARGS:--test.timeout 60m --junit-output=exported-artifacts/cdi.junit.xml}"
```

Comment:
Im not sure why we need these 2 different executions,
and not just one

Reply:
I don't know neither, currently result of test compilation are two test binaries ...

Reply:
it was only because in test_suit_test, `BeforeTestSuitSetup` function created some uncessary pv/pvc, sometimes create cdi pvc may failed, so I have to branchout a new TestSuit. but I have a pr to remove those things https://github.com/kubevirt/kubevirt-ansible/pull/357, but I just not sure if which way we would like to adopt.